ui/packages/catalog-admin-module-sources/src/main/webapp/js/model/Status.js

```diff
@@ -13,19 +13,24 @@
  *
  **/
 /*global define*/
-define(['backbone'], 
+define(['backbone'],
 function (Backbone) {
 
+    var statusEndpoint = "../jolokia/exec/org.codice.ddf.catalog.admin.poller.AdminPollerServiceBean:service=admin-source-poller-service/sourceStatus/";
     var Status = {};
 
     Status.Model = Backbone.Model.extend({
-        url: "../jolokia/exec/org.codice.ddf.catalog.admin.poller.AdminPollerServiceBean:service=admin-source-poller-service/sourceStatus/",
-            model: Status.Model,
+        url: undefined,
+        model: Status.Model,
 
-            initialize: function(pid) {
-                this.url += pid;
-            }
-        });
+        initialize: function(pid) {
```

Comment:
I think this model could be updated to use http://backbonejs.org/#Model-urlRoot

That in combination with setting id to pid should set the url correctly.

Reply:
👍 Didn't know about that. I can change it